src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,121 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int capacity;
+    private int size;
+    private Entry<K, V>[] table;
+
+    private static class Entry<K,V> {
+        final K key;
+        V value;
+        Entry<K,V> next;
+
+        Entry(K key, V value, Entry<K,V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
+
+    public MyHashMap() {
+        size = 0;
+        capacity = DEFAULT_INITIAL_CAPACITY;
+        table = new Entry[capacity];
+    }
+
+    private int hash(K key) {
+        return  Math.abs(key.hashCode()) % table.length;
+    }
+
+    private int countThreshold(int capacity) {
+        return Math.round(capacity * DEFAULT_LOAD_FACTOR);
```

Comment:
this one-line method isn't very useful
we can write in line 101: size >= capacity * DEFAULT_LOAD_FACTOR